src/main/webapp/app/entities/lecture-unit/attachmentUnit.model.ts

```diff
@@ -6,8 +6,17 @@ export class AttachmentUnit extends LectureUnit {
     public description?: string;
     public attachment?: Attachment;
     public slides?: Slide[];
+    public pyrisIngestionState?: IngestionState;
 
     constructor() {
         super(LectureUnitType.ATTACHMENT);
     }
 }
+
+export enum IngestionState {
```

Comment:
Why do you have more values here than in the server version of IngestionState?

Reply:
this is for the lectureUnits and the lectures together.